rfq/negotiator_test.go

```diff
@@ -0,0 +1,349 @@
+package rfq
+
+import (
+	"testing"
+	"time"
+
+	"github.com/lightninglabs/taproot-assets/asset"
+	"github.com/lightninglabs/taproot-assets/fn"
+	"github.com/lightninglabs/taproot-assets/rfqmath"
+	"github.com/lightninglabs/taproot-assets/rfqmsg"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+// testCaseIncomingSellAccept is a test case for the handling of an incoming
+// sell accept message.
+type testCaseIncomingSellAccept struct {
+	// Name is the name of the test case.
+	Name string
```

Comment:
nit: do all of these fields need to be exported? Since the main struct isn't?

Reply:
I'll make them all non-exported.